api/taskcluster/webhook.go

```diff
@@ -28,7 +28,7 @@ const flagPendingChecks = "pendingChecks"
 var (
 	// This should follow https://github.com/web-platform-tests/wpt/blob/master/.taskcluster.yml
```

Comment:
This file no longer defines the types of tests, if that is what this comment is meant to imply

Reply:
```suggestion
	// This should follow https://github.com/web-platform-tests/wpt/blob/master/tools/ci/tc/tasks/test.yml
```